### DISCUSSION TITLE
How to crop an image after you already cropped another before

### DISCUSSION THREAD

**Discussion by MathiasOlivera24** [ValentinH/react-easy-crop#382] [Q&A]
Hi, I'm trying to give the user the option to change the image once they already selected one. No problem I thought, I'll just change the image on my parent component, which I'm passing as a prop to the cropper. this seemed to have worked because the new image is displayed, but the problem is that the cropped result in the onCropComplete function is the old image.
If the image hadn't changed, I wouldn't have written this, but the new image shows up fine.

thanks

**Comment by ValentinH** [comment]
Are you able to share a reproduction of this unexpected behavior?
